docs/generated/redact_safe.md

```diff
@@ -29,6 +29,7 @@ pkg/roachpb/metadata.go | `ReplicaType`
 pkg/roachpb/metadata.go | `StoreID`
 pkg/roachpb/method.go | `Method`
 pkg/roachpb/tenant.go | `TenantID`
+pkg/roachpb/tenant.go | `TenantName`
```

Comment:
are we sure this is safe for reporting? looking briefly in this file, I think this would be the only string here that is generated by the user, which is why I ask..

Reply:
indeed, that doesn't seem right

Reply:
good catch, it was a copy pasta of TenantID that is marked `RedactSafe`. Removing.